assets/src/blocks/ActionsList/index.js

```diff
@@ -1,37 +1,31 @@
-const BLOCK_NAME = 'planet4-blocks/actions-list';
-
 // Register the ActionsList block.
 export const registerActionsList = () => {
   const {registerBlockVariation} = wp.blocks;
   const {__} = wp.i18n;
 
-  // We need to make sure that we have access to the Planet 4 options,
-  // to know which post type to use in the query depending on the new IA.
-  const interval = setInterval(() => {
-    if (typeof window.p4ge_vars === 'undefined') {
-      return;
-    }
+  const IS_NEW_IA = window.p4ge_vars.planet4_options.new_ia === 'on';
```

Comment:
I know this isn't in the ticket scope, but maybe we should move these variables to our master-theme `p4_vars`? It won't work if the plugin is deactivated 🤔